pkg/pillar/hypervisor/kvm.go

```diff
@@ -292,7 +302,7 @@ func newKvm() Hypervisor {
 			domains:      map[string]int{},
 			devicemodel:  "virt",
 			dmExec:       "qemu-system-aarch64",
-			dmArgs:       []string{"-display", "none", "-daemonize", "-S", "-no-user-config", "-nodefaults", "-no-shutdown", "-serial", "pty"},
+			dmArgs:       []string{"-display", "none", "-daemonize", "-S", "-no-user-config", "-nodefaults", "-no-shutdown", "-no-reboot", "-serial", "pty"},
 			dmCPUArgs:    []string{"-cpu", "host"},
```

Comment:
Wait @adarsh-zededa -- I don't understand how no-reboot would work in legitimate cases of VM rebooting. Can you explain?

Reply:
@rvs , I need to do some more changes regarding panic handling. I removed panic handling part from this PR and will be raising a separate PR for that. Will carry on this discussion in that PR.